.github/actions/env/action.yml

```diff
@@ -24,22 +24,23 @@ runs:
         auto-update-conda: true
         python-version: ${{ inputs.python-version }}
 
-    - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) }}
-      name: Install dependencies
+    - name: Install curl via conda
       shell: bash -l {0}
-      run: |
-        conda install curl -c conda-forge
-        conda install pytorch cpuonly -c pytorch-${{ inputs.pytorch-version }}
-        pip install .[dev,x]
+      run: conda install curl -c conda-forge
 
+    - name: Install Kornia dev
+      shell: bash -l {0}
+      run: pip install -e .[x,dev]
+
+    - if: ${{ contains(fromJson('["lts", "nightly"]'), inputs.pytorch-version ) }}
```

Comment:
i think the order should be 
1. install torch
2. install kornia + deps

Reply:
what's happening now is that first pip install install latest torch version and skip line 35